Decred Wallet/table_view_cell/AccountDataCell.swift

```diff
@@ -58,39 +56,37 @@ class AccountDataCell: UITableViewCell, AccountDetailsCellProtocol {
         
         if BuildConfig.IsTestNet {
             labelHDPathValue.text = "\(GlobalConstants.Strings.TESTNET_HD_PATH) \(account.Number)'"
-        }else {
+        } else {
             labelHDPathValue.text = "\(GlobalConstants.Strings.MAINNET_HD_PATH) \(account.Number)'"
         }
         
-        if (account.Number == INT_MAX) {
+        if account.Number == INT_MAX {
             defaultAccount.setOn(false, animated: false)
             defaultAccount.isEnabled = false
             hideAcount.setOn(false, animated: false)
             hideAcount.isEnabled = false
-        }else{
-            let hidden = UserDefaults.standard.bool(forKey: "hidden\(account.Number)")
-            if (hidden){
+        } else {
+            if account.isDefault {
```

Comment:
Why are there multiple checks for `if account.isDefault` in this block? Lines 69 and 76.
I think this first one should be `if account.isHidden`?